controllers/C_BillingContact.class.php

```diff
@@ -12,7 +12,7 @@ function __construct($template_mod = "general")
         parent::__construct();
         $this->icompanies = array();
         $this->template_mod = $template_mod;
-        $this->assign("FORM_ACTION", $GLOBALS['webroot']."/controller.php?" . $_SERVER['QUERY_STRING']);
+        $this->assign("FORM_ACTION", $GLOBALS['webroot']."/controller.php?" . attr($_SERVER['QUERY_STRING']));
```

Comment:
Explanation - 
In most of smarty template there is forms in sach style-
![image](https://user-images.githubusercontent.com/17809866/30636007-dc031af0-9dfc-11e7-9806-2d4f65f9960e.png)

This enable attack like -
![image](https://user-images.githubusercontent.com/17809866/30636106-42173ee8-9dfd-11e7-8eb9-bfb307fdf822.png)

In the Explorer -
 https://stackoverflow.com/questions/4581837/is-serverquery-string-safe-from-xss

Reply:
Usually it's best to html escape downstream where you output the string to html.
However, I think this strategy is ok in this case, since it will avoid needing to html escape $GLOBALS['webroot'] downstream.